src/extension/utils/createAndOpenConfFile.ts

```diff
@@ -11,11 +12,17 @@ type ConfFile = {
   staging?: string
   cache?: string
   msg?: string
-} & Record<string, boolean | string>
+} & Record<string, boolean | string | number>
 
 function setAdditionalSetting(val?: string) {
+  console.log(val)
   if (val === 'true' || !val) return true
   if (val === 'false') return false
+  if (/^[0-9]+$/.exec(val)) return Number(val)
+  const objectRegex = /^{(".+":".+")(,".+":".+")*}/g
```

Comment:
So, do you expect it to be a string to string "mapping", with at least a single key-value pair, correct?
Using regex for a JSON object is a bit tricky. Multiple valid JSON formats don't match the above pattern.

Reply:
it should be a specific object pattern, looks like:
`{"name":"value", "otherName": "other_value"}` etc
as I understood this is they way we are receiving object inputs.